hibernate-search-plugins/org.jboss.tools.hibernate.search/src/org/jboss/tools/hibernate/search/HSearchConsoleConfigurationPreferences.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.tools.hibernate.search;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hibernate.console.ConsoleConfiguration;
+import org.hibernate.console.KnownConfigurations;
+
+public class HSearchConsoleConfigurationPreferences {
+	
+	private static final String DEFAULT_HIBERNATE_SEARCH_VERSION = "4.3";
```

Comment:
Sure. Up to you which versions you think you can maintain. I'm simply suggesting that including many old versions is probably not going to be helpful, yet then it's possible that some people will expect them to be maintained.

I would suggest starting from Hibernate Search 5.5.5 and then onwards.